lib/hooks/queries/useTotalLiquidity.ts

```diff
@@ -1,42 +1,40 @@
 import { useQuery } from "@tanstack/react-query";
 import Decimal from "decimal.js";
 import { useSdkv2 } from "../useSdkv2";
+import { ZTG } from "@zeitgeistpm/sdk-next";
 
 export const rootKey = "total-liquidity";
 
-export const useTotalLiquidity = (options: { enabled: boolean }) => {
+const calcLiqudity = (
+  assets: { price: string | number; amountInPool: any }[],
+) => {
+  return assets.reduce((total, asset) => {
+    if (!asset.price || !asset.amountInPool) {
+      return total;
+    }
+    const price = new Decimal(asset.price);
+    return total.plus(
+      new Decimal(price.div(ZTG)).mul(new Decimal(asset.amountInPool)),
+    );
+  }, new Decimal(0));
+};
+
+export const useTotalLiquidity = () => {
   const [sdk, id] = useSdkv2();
 
-  const { data: pools } = useQuery(
+  return useQuery(
     [id, rootKey],
-    () => sdk.model.swaps.listPools({}),
-    {
-      enabled: options.enabled && Boolean(sdk),
-    },
-  );
-
-  const { data: saturatedIndex } = useQuery(
-    [id, rootKey, "saturated-index"],
     async () => {
-      return sdk.model.swaps.saturatedPoolsIndex(pools);
+      const pools = await sdk.model.swaps.listPools({});
+      const total =
+        pools?.reduce((acc, pool) => {
+          return acc.plus(calcLiqudity(pool.assets as any));
```

Comment:
This won't be quite right now that we have different base assets in the pool. We will need to convert those to be denominated in ZTG. Happy to move this to another ticket though as it was already like this